### PR TITLE
Fixes #18196: Rtf exit code is 0 when the system of a platform is unknow

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/18196

### CODE REVIEW

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
Commit modified

**Review by peckpeck — APPROVED** [review]

File: lib/host.py [hunk 1]
```diff
@@ -53,7 +54,8 @@ def start(self):
           idle = random.randint(5, 10)
           print("Lock not available, waiting %s seconds"%idle)
           time.sleep(idle)
-    ret = os.system(proxy + "vagrant up " + self.hostid + " --provider="+self.provider)
+    command = proxy + "vagrant up " + self.hostid + " --provider="+self.provider
+    ret = subprocess.call(command.split(" "))
```
Comment (peckpeck) [inline lib/host.py:58]: os.system keeps stdout and stderr to terminal, whereas subprocess call removes it

Comment (Fdall) [reply lib/host.py:58]: There is no need for this whe you use the call method, see https://docs.python.org/fr/3/library/subprocess.html#subprocess.call
It does not removes stdout and stderr by default

**Pull request merged**